dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/messages/messages.properties

```diff
@@ -150,14 +150,15 @@ MoveProjectTimeout=A timeout occurred while moving the {0} project from {1} to {
 
 ProjectDeployedDialogShell=Manage Deployments
 ProjectDeployedDialogTitle=Project Already Deployed
-ProjectDeployedDialogMessage=The project at location {0} is already deployed and enabled on other connections. It is not recommended to have the same project on multiple connections.
-ProjectDeployedDialogGroupLabel=Choose the behaviour for existing enabled deployments
+ProjectDeployedDialogMessage=The project at location {0} is already deployed and enabled on other connections.
+ProjectDeployedDialogInstructions=The project is already deployed on connections: {0}. Remove, disable or maintain the existing deployments of this project. Click Cancel to go back and select a different project.
```

Comment:
- What about, "The project is already deployed on these connections:"?
- Also, please add an Oxford comma: "Remove, disable, or maintain the..." This matches the style we use in the docs.

Reply:
Done.